src/lib/chips/chips.scss

```diff
@@ -1,27 +1,70 @@
-$mat-chip-vertical-padding: 8px;
-$mat-chip-horizontal-padding: 12px;
+$mat-chip-font-size: 13px;
+$mat-chip-line-height: 16px;
+$mat-chip-border-width: 2px;
 
-$mat-chips-chip-margin: $mat-chip-horizontal-padding / 4;
+$mat-chip-vertical-padding: 8px - $mat-chip-border-width;
+$mat-chip-horizontal-padding: 12px - $mat-chip-border-width;
+
+$mat-chip-margin: ($mat-chip-horizontal-padding / 4);
+
+$mat-chip-remove-margin: $mat-chip-line-height * 2;
+$mat-chip-remove-icon-offset: 3px;
+$mat-chip-remove-size: 24px;
+$mat-chip-remove-font-size: 18px;
 
 .mat-chip-list-wrapper {
+
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
   align-items: flex-start;
+
+  .mat-chip:not(.mat-basic-chip) {
```

Comment:
Maybe for a follow-up PR (can add a TODO), but I'd like to get rid of all of the `:not(.mat-basic-chip)`. It raises the specificity such that you can't just target `.mat-chip`. Would be better to target a CSS class that is only applied when the basic chip directive isn't applied.